tests/test_browser.py

```diff
@@ -64,7 +64,7 @@ class browser(BrowserCore):
   def setUpClass(self):
     super(browser, self).setUpClass()
     self.browser_timeout = 20
-    self.test_wasm_pthreads = os.environ.get('TEST_WASM_PTHREADS', '0') == '1'
```

Comment:
maybe add a comment here? it's less obvious what's going on if an env var for testing wasm pthreads sets "also wasm"..

Reply:
Oh, that's a good point, I forgot that we made the class var completely generic. Let me just change the env var for now. We should enable wasm browser tests by default. Although I guess we will also get that for free when we switch to -s WASM=1 by default?

Reply:
...and I just discovered that I did that too, with the `EMCC_BROWSER_ALSO_WASM` env var. So this line is just a leftover and can be removed.